File: Java/Logic - 2/makeBricks.java
public boolean makeBricks(int small, int big, int goal) {

    int maxBig = goal / 5;

    if(big > maxBig) {

        goal = goal - maxBig * 5;

    }

    else {

        goal = goal - big * 5;

    }

    if(small >= goal) {

        return true;

    }

    else {

        return false;

    }

}

File: Python/Logic - 2/make_bricks.py
def make_bricks(small, big, goal):
    
    big_req = goal//5
    
    if big_req >= big:
        
        goal -= big * 5
        
    else:
        
        goal -= big_req * 5
        
    if goal >= small:
        
        goal -= small
        
    else:
        
        goal = 0
        
    return goal <= 0